docker-library/apache-php-mysql/entrypoint.sh

```diff
@@ -84,6 +92,9 @@ test ! -d "$MARIADB_LOG_DIR" && echo "INFO: $MARIADB_LOG_DIR not found. creating
 chown -R mysql:mysql $MARIADB_LOG_DIR
 echo "Starting local MariaDB ..."
 start_mariadb
+# create /home/site/wwwroot for local machine
+test ! -d "$APPHOME" && echo "INFO: $APPHOME not found. creating ..." && mkdir -p "$APPHOME"
```

Comment:
I think this test is very important so that we should put it just below update_settings